src/samples/compression/EntryPointCompression.sol

```diff
@@ -0,0 +1,104 @@
+            mstore8(add(len, 11), 32)
+            mstore(add(ptr, 0x0180), 0x000000000000000000000000000000000000000000000000000000000000000d)
+            mstore8(add(len, 12), 32)
+            mstore(add(ptr, 0x01a0), 0x000000000000000000000000000000000000000000000000000000000000000e)
+            mstore8(add(len, 13), 32)
+            mstore(add(ptr, 0x01c0), 0x000000000000000000000000000000000000000000000000000000000000000f)
+            mstore8(add(len, 14), 32)
+            mstore(add(ptr, 0x01e0), 0x4bf114ff00000000000000000000000000000000000000000000000000000000)
+            mstore8(add(len, 15), 4)
+            mstore(add(ptr, 0x0200), 0x7cfc1a5d00000000000000000000000000000000000000000000000000000000)
+            mstore8(add(len, 16), 4)
+            mstore(add(ptr, 0x0220), 0xf551237e00000000000000000000000000000000000000000000000000000000)
+            mstore8(add(len, 17), 4)
+            mstore(add(ptr, 0x0240), 0x0000000000000000000000000000000000000000000000000000000000000020)
+            mstore8(add(len, 18), 32)
+            mstore(add(ptr, 0x0260), 0x0000000000000000000000000000000000000000000000000000000000000060)
+            mstore8(add(len, 19), 32)
+            mstore(add(ptr, 0x0280), 0x0000000000000000000000000000000000000000000000000000000000000080)
+            mstore8(add(len, 20), 32)
+            mstore(add(ptr, 0x02a0), 0x0000000000000000000000000000000000000000000000000000000000000041)
+            mstore8(add(len, 21), 32)
+            mstore(add(ptr, 0x02c0), 0x0000000000000000000000003aa5ebb10dc797cac828524e59a333d0a371443c)
+            mstore8(add(len, 22), 32)
+            /*
+            mstore(add(ptr, 0x02e0), 0x0000000000000000000000000000000000000000000000000000000000000000)
+            mstore8(add(len, 23), 32)
+            mstore(add(ptr, 0x0300), 0x0000000000000000000000000000000000000000000000000000000000000000)
+            mstore8(add(len, 24), 32)
+            mstore(add(ptr, 0x0320), 0x0000000000000000000000000000000000000000000000000000000000000000)
+            mstore8(add(len, 25), 32)
+            mstore(add(ptr, 0x0340), 0x0000000000000000000000000000000000000000000000000000000000000000)
+            mstore8(add(len, 26), 32)
+            mstore(add(ptr, 0x0360), 0x0000000000000000000000000000000000000000000000000000000000000000)
+            mstore8(add(len, 27), 32)
+            mstore(add(ptr, 0x0380), 0x0000000000000000000000000000000000000000000000000000000000000000)
+            mstore8(add(len, 28), 32)
+            mstore(add(ptr, 0x03a0), 0x0000000000000000000000000000000000000000000000000000000000000000)
+            mstore8(add(len, 29), 32)
+            mstore(add(ptr, 0x03c0), 0x0000000000000000000000000000000000000000000000000000000000000000)
+            mstore8(add(len, 30), 32)
+            mstore(add(ptr, 0x03e0), 0x0000000000000000000000000000000000000000000000000000000000000000)
+            mstore8(add(len, 31), 32)
+            */
```

Comment:
```suggestion
```

Reply:
I've kept this commented to show that other implementations can take advantage of more entries

Reply:
Ohhh got it